							# Empty request
---
course_map silicon_valley	# Map of the obstacle course